src/components/Dashboard/ProprietorPartnersListSm.vue

```diff
@@ -20,46 +20,52 @@
 
         <v-expansion-panel-content>
           <v-list class="pt-0 pb-0">
-            <v-list-item class="email-address-list-item" v-if="party.officer.email">
+            <v-list-item class="email-address-list-item">
               <v-list-item-content>
                 <v-list-item-title class="mb-2 address-title">Email Address</v-list-item-title>
-                <v-list-item-subtitle>{{ party.officer.email }}</v-list-item-subtitle>
+                <v-list-item-subtitle>{{ party.officer.email || 'Not Entered'}}</v-list-item-subtitle>
               </v-list-item-content>
             </v-list-item>
 
-            <v-list-item class="delivery-address-list-item" v-if="party.deliveryAddress">
+            <v-list-item class="delivery-address-list-item">
               <v-list-item-content>
                 <v-list-item-title class="mb-2 address-title">Delivery Address</v-list-item-title>
                 <v-list-item-subtitle>
-                  <ul class="address-subtitle pre-line">
+                  <ul class="address-subtitle pre-line" v-if="party.deliveryAddress">
                     <li class="address-line1">{{ party.deliveryAddress.streetAddress }}</li>
                     <li class="address-line2">{{ party.deliveryAddress.streetAddressAdditional }}</li>
                     <li class="address-line3">{{ party.deliveryAddress.addressCity }}
                                               {{ party.deliveryAddress.addressRegion }}
                                               {{ party.deliveryAddress.postalCode }}</li>
                     <li class="address-line4">{{ getCountryName(party.deliveryAddress.addressCountry) }}</li>
                   </ul>
+                  <ul class="address-subtitle pre-line" v-else>
+                    <li class="address-line1">Not Entered</li>
```

Comment:
Same as above: please use new classname, and also add a unit test or 2 😃

Reply:
I added new classes and 2 unit tests for the two related components.